crates/swc/tests/tsc-references/doWhileBreakStatements.2.minified.js

```diff
@@ -1,4 +1,2 @@
 //// [doWhileBreakStatements.ts]
-FOUR: for(;;)FIVE: for(;;)break FOUR;
 for(;;);
-for(;;)for(;;);
```

Comment:
Original implementation would transform
```js
SEVEN: do do do break SEVEN;
while (true);
while (true);
while (true);
```
into
```js
for(;;)for(;;);
```
but it's actually a loop that immediately break.